starter-code/src/viking.js

```diff
@@ -1,11 +1,68 @@
 // Soldier
-function Soldier() {}
+function Soldier(healthArg, strengthArg) {
+
+        this.health = healthArg;
+        this.strength = strengthArg;
```

Comment:
Here, you properly define the health and strength characteristics, but this is not how you should define the methods. The way you did it works, but there's a more "organized" way of doing it:

```javascript
function Soldier(healthArg, strengthArg) {
    this.health = healthArg;
    this.strength = strengthArg;
}

Soldier.prototype.attack = function() {
    return this.strength;
}

Soldier.prototype.receiveDamage = function( damageArg ) {
    this.health -= damageArg;
}
```